src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1521,20 +1555,21 @@ def forward(self, inputs, targets):
                     context.update_context(metrics_dict=test_metrics_dict)
                     self.phase_callback_handler.on_test_loader_end(context)
 
+                    self._write_scalars_to_logger(metrics=test_metrics_dict, epoch_values=None, epoch=1 + epoch, inference_time=test_inf_time, tag="Test")
```

Comment:
Feels like we should simply inline _write_scalars_to_logger method as all it does - calling a logger.add_scalars a couple of times here. @shaydeci maybe you have better context on how sg_logger evolved over time, WDYT the suggested change?